src/components/filePanoItem.ts

```diff
@@ -58,7 +58,7 @@ export class FilePanoItem extends PanoItem {
           x_align: ActorAlign.FILL,
           x_expand: true,
         });
-        uriLabel.clutter_text.ellipsize = EllipsizeMode.MIDDLE;
+        uriLabel.clutter_text.ellipsize = EllipsizeMode.START;
```

Comment:
Surely, the start and the end of the path are important. But sometimes on long paths, the file name is not understandable what file are you copying. I think if there was a mechanism having only the file name shown without the full path, but showing the full path on hover it would have been better.